ptx/sec_deriv_prodquot.ptx

```diff
@@ -518,10 +510,12 @@
     <statement>
       <p>
         Find the derivatives of the following functions.
-
         <ol>
-          <li><m>f(x) = \dfrac{1}{x}</m></li>
-
+          <li>
```

Comment:
Maybe `<li><m>f(x)=\frac{1}{x}</m></li>` is best? Without `<p>` it will insert displaystyle automatically.

Reply:
The issue is that the other list item in this has a `p`. If one has a `p` and the other does not, I saw something bad wrt formatting in output. Can't recall if it was HTML or PDF, but somewhere the `p` induces some extra vertical spacing, and it looked off if one item had that but not the other. Maybe the other item doesn't need a `p`. If it is important to change this, we should look at the output in both formats before committing.

Reply:
Ah, that could be it.
I think that's a known problem but can't recall if it's been brought up with Rob. (I think yes.) Without the `p` there sometimes isn't enough vertical space.

This makes sense then